src/main/java/com/owncloud/android/lib/resources/files/DownloadFileRemoteOperation.java

```diff
@@ -175,8 +175,6 @@ private int downloadFile(OwnCloudClient client, File targetFile) throws IOExcept
                 client.exhaustResponse(getMethod.getResponseBodyAsStream());
             }
 
-        } catch (Exception e) {
```

Comment:
What happens if another Exception is thrown? (if any)

Reply:
So according to my usage search this private `downloadFile()` method is only ever called from `run()` in the same class. There, the call is already wrapped in a try/catch that catches all `Exception`s, but there it gets handled, while here it just gets swallowed.

Reply:
My concern was that any other exception might lead to a crash, but now I see that this is handled then in run(), as you wrote :+1: